src/main/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfiguration.java

```diff
@@ -43,24 +44,51 @@ private ObjectMapper createMapper(DocumentLimitsConfig documentLimitsConfig) {
-        .enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS)
+    JsonMapper mapper =
+        JsonMapper.builder(jsonFactory)
+            // important for retaining number accuracy!
+            .enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS)
+
+            // case-insensitive enums, so "before" will match to "BEFORE" in an enum
+            .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
 
-        // case-insensitive enums, so "before" will match to "BEFORE" in an enum
-        .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
+            // Verify uniqueness of JSON Object properties
+            .enable(StreamReadFeature.STRICT_DUPLICATE_DETECTION)
 
-        // Verify uniqueness of JSON Object properties
-        .enable(StreamReadFeature.STRICT_DUPLICATE_DETECTION)
+            /* 12-Dec-2023, tatu: Must not force use of Plain notation because
+             *  that caused: https://github.com/stargate/jsonapi/issues/726
+             *  where we can insert numbers that are not round-trippable
+             *  as they get expanded from more compact Engineering/Scientific
+             *  notation to longer Plain notation; effectively creating "unreadable"
+             *  Documents.
+             */
+            .disable(StreamWriteFeature.WRITE_BIGDECIMAL_AS_PLAIN)
+            .build();
 
-        /* 12-Dec-2023, tatu: Must not force use of Plain notation because
-         *  that caused: https://github.com/stargate/jsonapi/issues/726
-         *  where we can insert numbers that are not round-trippable
-         *  as they get expanded from more compact Engineering/Scientific
-         *  notation to longer Plain notation; effectively creating "unreadable"
-         *  Documents.
-         */
-        .disable(StreamWriteFeature.WRITE_BIGDECIMAL_AS_PLAIN)
-        .build();
+    mapper.addHandler(
+        new DeserializationProblemHandler() {
+          @Override
+          public JavaType handleUnknownTypeId(
+              DeserializationContext ctxt,
+              JavaType baseType,
+              String subTypeId,
+              TypeIdResolver idResolver,
+              String failureMsg)
+              throws JsonApiException {
+            //            interface io.stargate.sgv2.jsonapi.api.model.command.NamespaceCommand ->
+            // NamespaceCommand
+            //            interface io.stargate.sgv2.jsonapi.api.model.command.CollectionCommand ->
+            // CollectionCommand
+            //            interface io.stargate.sgv2.jsonapi.api.model.command.GeneralCommand ->
+            // GeneralCommand
+            final String rawCommandClassString = baseType.getRawClass().toString();
+            final String baseCommand =
+                rawCommandClassString.substring(rawCommandClassString.lastIndexOf('.') + 1);
+            throw new JsonApiException(
+                ErrorCode.NO_COMMAND_MATCHED,
+                String.format("No '%s' command found as %s", subTypeId, baseCommand));
```

Comment:
nit: surround second `%s` in (double?) quotes.

Reply:
This is an example. I am thinking that CollectionCommand is an actually word existing in JSONAPI. 
unknownCommand is a word provided by user, so I quote it?
```
{
    "errors": [
        {
            "message": "No 'unknownCommand' command found as CollectionCommand",
            "errorCode": "NO_COMMAND_MATCHED"
        }
    ]
}
```

Reply:
I think that In general, anything that is identifier in system/code, should be quoted. But not a big deal.

Reply:
solved